recipes/vgan/meta.yaml

```
@@ -1,24 +1,27 @@
 {% set name = "vgan" %}
-{% set version = "2.0.2" %}
+{% set version = "3.0.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://github.com/grenaud/vgan/releases/download/v{{ version }}/vgan
-  sha256: 4340e61ff3c4ca8f8eca388604cc004d23d9399a4df860116f3cab342599d14e
+  sha256: 7a8fb93618fc0efe49f33e998a4f46fbd004c63c3326fa9c0e414940418c1c7b
 
 build:
   number: 0
   skip: true  # [osx]
+  run_exports:
+    - {{ pin_subpackage('vgan', max_pin="3.0.0") }}
```

```suggestion
    - {{ pin_subpackage('vgan', max_pin="x") }}
```

or x.x depending on how stable the API is